osu.Game.Rulesets.Mania/ManiaMobilePlayStyle.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+using osu.Game.Localisation;
+
+namespace osu.Game.Rulesets.Mania
+{
+    public enum ManiaMobilePlayStyle
+    {
+        [LocalisableDescription(typeof(RulesetSettingsStrings), nameof(RulesetSettingsStrings.TouchableColumns))]
+        TouchableColumns,
+
+        [LocalisableDescription(typeof(RulesetSettingsStrings), nameof(RulesetSettingsStrings.TouchControls))]
+        TouchControls,
+
+        [LocalisableDescription(typeof(RulesetSettingsStrings), nameof(RulesetSettingsStrings.ExtendedColumns))]
```

Comment:
Does this mean "landscape with expanded columns"?

Let's call it `Landscape` / "Landscape (expanded columns)"

Reply:
https://github.com/ppy/osu/pull/32314/commits/b64e69d581a08d0be9d5e705f9bb1078faf3050f